lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM/SharedToDotOperandMMAv2.cpp

```diff
@@ -2,7 +2,7 @@
 
 using namespace mlir;
 
-using ValueTable = std::map<std::pair<unsigned, unsigned>, Value>;
+using ValueTable = std::map<std::tuple<unsigned, unsigned, unsigned>, Value>;
```

Comment:
So the first dimension is the batch dimension?

Reply:
can we use std::array<unsigned, 3> here?